Source/WebCore/rendering/line/WordTrailingSpace.h

```diff
@@ -45,15 +46,16 @@ struct WordTrailingSpace {
         if (m_state == WordTrailingSpaceState::Initialized)
             return m_width;
 
-        auto& font = m_style.fontCascade();
-        m_width = font.width(RenderBlock::constructTextRun(span(space), m_style), &fallbackFonts) + font.wordSpacing();
+        CheckedRef style = m_style.get();
+        CheckedRef font = style->fontCascade();
+        m_width = font->width(RenderBlock::constructTextRun(span(space), style), &fallbackFonts) + font->wordSpacing();
         m_state = WordTrailingSpaceState::Initialized;
         return m_width;
     }
 
 private:
     enum class WordTrailingSpaceState { Uninitialized, Initialized };
-    const RenderStyle& m_style;
+    CheckedRef<const RenderStyle> m_style;
```

Comment:
We should mark this data member as `const`, then you'll no longer need `style` stack variables.